gufe/components/solventcomponent.py

```diff
@@ -22,39 +56,94 @@ class SolventComponent(Component):
     by specific MD engine methods.
     """
     _smiles: str
-    _positive_ion: Optional[str]
-    _negative_ion: Optional[str]
+    _positive_ion: str
+    _negative_ion: str
     _neutralize: bool
     _ion_concentration: unit.Quantity
 
     def __init__(self, *,  # force kwarg usage
                  smiles: str = 'O',
+                 num_solvent: Optional[int] = None,
                  positive_ion: str = 'Na+',
                  negative_ion: str = 'Cl-',
                  neutralize: bool = True,
-                 ion_concentration: unit.Quantity = 0.15 * unit.molar):
+                 ion_concentration: Optional[unit.Quantity] = 0.15 * unit.molar,
+                 solvent_padding: Optional[unit.Quantity] = 1.2 * unit.nanometer,
+                 solvent_density: Optional[unit.Quantity] = None,
```

Comment:
this is probably worth including, as it *does* affect the reduced potential

Reply:
I'm not sure what you mean here - density is a function of N and volume, everything here defines both of those things.